docs/en_US/Trials.md

```diff
@@ -125,6 +125,28 @@ In the YAML configure file, you need to set *useAnnotation* to true to enable NN
 useAnnotation: true
 ```
 
+
+## Where are my trials?
+
+In NNI, every trial has a dedicated directory for them to output their own data. In each trial, an environment variable called `NNI_OUTPUT_DIR` is exported. Under this directory, you could find each trials code, data and other possible log. In addition, each trial's log will be re-directed to a file name `trial.log` under that directory.
+
+Besides, trial's converted code is in another temporary directory if NNI Annotation is used. You can check that in a file named `run.sh` under the directory indicated by `NNI_OUTPUT_DIR`. The second line of this file will change directory to the actual directory where code is located. Below is an example of `run.sh`:
```

Comment:
In local mode, trial.log and run.sh is in `NNI_OUTPUT_DIR`. But in remote, pai and kubeflow mode, `NNI_OUTPUT_DIR` specifies the trial's output directory in remote machines,  and NNI may not copy run.sh and code files to this folder, and trial.log is transmitted to local machine in ~/nni/experiments/{expid}/log/trial_{id}.log.

Reply:
I know, but it's really hard to say cuz there's no such easily-expressed rules clear enough to a new user....
So I might put a constrain to the above explanation that it is only applicable in local mode. And in other modes, this environmental variable only stands for trial's output directory ?